educator/src/Dscp/Educator/DB/Queries.hs

```diff
@@ -354,6 +357,25 @@ createPrivateBlock delta = runMaybeT $ do
 
     return hdr
 
+dumpNonChainedTransactions
+    :: (DBM m, HasCtx ctx m '[KeyResources EducatorNode])
+    => Maybe ATGDelta
+    -> DBT 'WithinTx m (Maybe PrivateBlockHeader)
+dumpNonChainedTransactions atgDelta = do
+    txs <- getAllNonChainedTransactions
+    createPrivateBlock txs atgDelta
+
+createPrivateBlockFromCert
+    :: (DBM m, HasCtx ctx m '[KeyResources EducatorNode])
+    => CertificateFullInfo -> DBT 'WithinTx m (Maybe PrivateBlock)
+createPrivateBlockFromCert cert = do
+    time <- toTimestamp <$> liftIO getCurrentTime
```

Comment:
Semantically, grade timestamp should represent the time when the grade has been issued. Maybe it's better to use `issueDate` here (like, pretending that grades has appeared at 00:00 on the day of diploma issue)?

Reply:
Okay, this makes sense.

Reply:
I would probably set to 12:00, or guys from London will see grades being issued a day before the certificate issue date (which is fine, but to avoid inconsistency and extra questions in future...)